src/themes/base/colors.ts

```diff
@@ -135,11 +143,40 @@ export const contextualColors: ContextualColors = {
   warning: naturalColors.yellow,
 }
 
-export const colors: ColorPalette = {
-  ...contextualColors,
+const emphasisAndNaturalColors: EmphasisColors & NaturalColors = {
   ...emphasisColors,
   ...naturalColors,
+}
+
+const lightBackgroundColors = ['teal', 'yellow']
```

Comment:
@kuzhelov agreed, what is your proposal?

add `background` as a key to the `ColorVariants` type?

Reply:
yes, essentially I would propose to reconsider props of the `ColorVariants` type, yes - and just add the only necessary to inline this information currently kept by this array. It may be a `background` prop, maybe other set (like `light` and `dark`, and this will allow us to properly apply colors on top of each other). 

There are several corner cases that we won't be able to cover with the naive approach, thus I would rather like to open a separate discussion around exact implementation - but generally, yes, would rather see this information as being part of the `ColorVariants` type.

Reply:
@kuzhelov got it, agreed it'd be a better way to do things and was thinking of something similar myself
@layershifter can you weigh in as well?

Reply:
Will keep a track of this in an issue together will the other proposals for improvements.